README.md

```diff
@@ -103,17 +120,17 @@ those dependencies.
 
 [venv]: http://chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
 
-```console
+```bash
 $ pip install -r requirements-dev.txt -r requirements.txt
 ```
 
 Then, point your environment to this directory:
```

Comment:
Omit filling words. Use a numbered list if you like. This is indicating a sequential order.

Reply:
I'm sorry but this is not in the scope of this PR. Feel free to open new story to fix this.